### PR TITLE
Add protocols to assets

### PR DESCRIPTION
**Pull request by yabirgb**
Added:

- aave-v1
- aave-v2
- compound
- yearn v1

To do so I made a query in the database to get the assets that contained in the name the protocol and then used:

```
def aave_entries():
    output = []
    for asset in AAVE_IDS.split('\n'):
        token = EthereumToken(asset.replace('_ceth_', ''))
        if 'v2' in token.name:
            protocol = 'aave-v2'
        else:
            protocol = 'aave'
        output.append(f'UPDATE ethereum_tokens SET protocol="{protocol}" where address="{token.ethereum_address}";')
        output.append(f'INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("{token.ethereum_address}", {token.decimals}, "{protocol}"); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("{asset}", "C", "{token.name}", "{token.symbol}", {token.started}, NULL, "{token.coingecko}", "{token.cryptocompare}", "{token.ethereum_address}");')

    return "\n".join(output)

def compound_entries():
    output = []
    protocol = 'compound'
    for asset in COMPOUND_IDS.split('\n'):
        token = EthereumToken(asset.replace('_ceth_', ''))
        
        output.append(f'UPDATE ethereum_tokens SET protocol="{protocol}" where address="{token.ethereum_address}";')
        output.append(f'INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("{token.ethereum_address}", {token.decimals}, "{protocol}"); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("{asset}", "C", "{token.name}", "{token.symbol}", {token.started}, NULL, "{token.coingecko}", "{token.cryptocompare}", "{token.ethereum_address}");')

    return "\n".join(output)

def yearnv1_entries():
    output = []
    protocol = 'yearn-v1'
    for asset in YEARN_V1:
        token = GlobalDBHandler().get_assets_with_symbol(asset, AssetType.ETHEREUM_TOKEN)[0]
        output.append(f'UPDATE ethereum_tokens SET protocol="{protocol}" where address="{token.ethereum_address}";')
        output.append(f'INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("{token.ethereum_address}", {token.decimals}, "{protocol}"); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("{asset}", "C", "{token.name}", "{token.symbol}", {token.started}, NULL, "{token.coingecko}", "{token.cryptocompare}", "{token.ethereum_address}");')

    return "\n".join(output)
```

### CODE REVIEW

**Comment by yabirgb** [comment]
Didn't want to create another PR so added a new commit with 2 more assets for tokenized coinbase tokens on FTX and bittrex

**Review by LefterisJP — COMMENTED** [review]
Hey thanks but as you can see below scripts are not perfect and lead to errors. I think the error in your case was your assumption that the name will have v2 for aave v2, while this is the case only for aave tokens that existed in v1, but if they did not exist in v1 then they don't have the suffix.

Also apparently we were missing some tokens.

I checked aave v1 tokens from https://docs.aave.com/developers/v/1.0/deployed-contracts/deployed-contract-instances. We include them all.

I checked aave v2 tokens from https://docs.aave.com/developers/deployed-contracts/deployed-contracts. We are missing:

- aBAL - https://etherscan.io/address/0x272F97b7a56a387aE942350bBC7Df5700f8a4576
- aRAI - https://etherscan.io/address/0xc9BC48c72154ef3e5425641a3c747242112a46AF
- aXSUSHI - https://etherscan.io/address/0xf256cc7847e919fac9b808cc216cac87ccf2f47a

You mis-classified as aave v1:

- aCRV - https://etherscan.io/address/0x8dAE6Cb04688C62d939ed9B68d32Bc62e49970b1
- aWETH - https://etherscan.io/address/0x030bA81f1c18d280636F32af80b9AAd02Cf0854e
- aGUSD - https://etherscan.io/address/0xD37EE7e4f452C6638c96536e68090De8cBcdb583

For compound I checked from here: https://compound.finance/docs#networks . And we are missing:

- cLINK - https://etherscan.io/token/0xface851a4921ce59e912d19329929ce6da6eb0c7
- cTUSD - https://etherscan.io/token/0x12392f67bdf24fae0af363c24ac620a2f67dad86
- cWBTC2 - https://etherscan.io/token/0xccf4429db6322d5c611ee964527d42e5d685dd6a  (not really sure what this is)


For yearn v1 I am not sure where to find a list. All vaults from `YEARN_VAULTS` in yearn/vaults.py are there though.

What about yearn v2?

File: updates/6/updates.sql [hunk 1]
```diff
@@ -303,4 +303,144 @@ INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingec
 INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("UNIDOWN", "W", "UNIDOWN", "UNIDOWN", 1603785600, NULL, NULL, "UNIDOWN", "UNIDOWN"); INSERT INTO common_asset_details(asset_id, forked) VALUES("UNIDOWN", NULL);
 *
 INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("UNIUP", "W", "UNIUP", "UNIUP", 1603785600, NULL, NULL, "UNIUP", "UNIUP"); INSERT INTO common_asset_details(asset_id, forked) VALUES("UNIUP", NULL);
+*
```
[the remark this answers is not in the document]
Comment (yabirgb) [reply updates/6/updates.sql:306]: No, if you check the last line it's not in green and has a `*`. It's for how git identifies the changes

**Comment by yabirgb** [comment]
@LefterisJP The yearn v2 already have their protocol set, same for curve

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**